components/preparing_workspace/preparing_workspace.tsx

```diff
@@ -233,7 +233,9 @@ export default function PreparingWorkspace(props: Props) {
             return;
         }
 
-        let team = currentTeam;
+        // In cloud instances created from portal,
+        // new admin user has a team in myTeams but not in currentTeam.
+        let team = (currentTeam || myTeams?.[0]);
```

Comment:
Are parenthesis required by lint?

Reply:
As the same validation is done in multiple places of `PreparingWorkspace` could be consolidated in a constant.

Reply:
I don't think they're required, but I left them since the linter didn't mind. I can change. Consolidating to a constant sounds good to me!